explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -457,14 +455,13 @@ object TargetTabContents extends TwoPanels:
       .useContext(AppContext.ctx)
       // Two panel state
       .useStateView[SelectedPanel](SelectedPanel.Uninitialized)
-      .useEffectWithDepsBy((props, _, state) => (props.focused, state.reuseByValue)) {
-        (_, _, selected) => (focused, _) =>
-          (focused, selected.get) match
-            case (Focused(Some(_), _, _), _)                    => selected.set(SelectedPanel.Editor)
-            case (Focused(None, Some(_), _), _)                 => selected.set(SelectedPanel.Editor)
-            case (Focused(None, None, _), SelectedPanel.Editor) =>
-              selected.set(SelectedPanel.Summary)
-            case _                                              => Callback.empty
+      .useEffectWithDepsBy((props, _, state) => props.focused) { (_, _, selected) => focused =>
```

Comment:
Weird, we don't want to run this if `state` changes? Not sure how this ends up producing the infinite loop

Reply:
Well, `state` is the same as `selected`, which we set in the method. So, we're both depending on it and setting it. I guess it usually `settles down` to a value, but the double clicking upsets that somehow. I should change `state` to `_`.

Reply:
I haven't seen a case where I think removing it has affected the behavior. But, I might be missing something. 

We actually do the same thing in `ObsTabContents`. I'll see if that still works if I remove it.